numpy-stubs/__init__.pyi

```diff
@@ -294,17 +296,23 @@ class _ArrayOrScalarCommon(
 _BufferType = Union[ndarray, bytes, bytearray, memoryview]
 
 class ndarray(_ArrayOrScalarCommon, Iterable, Sized, Container):
-    real: ndarray
-    imag: ndarray
+    @property
```

Comment:
The documentation for ``ndarray.real`` and ``ndarray.imag`` ([ref](https://numpy.org/doc/stable/reference/generated/numpy.real.html)) suggests that both getting and setting should be fine.